7_react/react1/week2/todo_list_app/src/components/todoList.jsx

```diff
@@ -0,0 +1,52 @@
+const TodoList = ({ todos, setTodos }) => {
+  const handleComplete = (todo) => {
+    setTodos(
+      todos.map((item) => {
+        if (item.id === todo.id) {
+          return { ...item, completed: !item.completed };
+        }
+        return item;
+      })
+    );
+  };
+  const handleDelete = ({ id }) => {
+    if (todos.length === 1) {
+      setTodos([]);
+      alert("No items left");
```

Comment:
The assignment requested that once there are no more todos left, the app should display "No items left" **instead** of the todos. This means that instead of an alert you should just render a simple `<p>` tag, in the case that the length of the todos is 0.